GDJS/Runtime/runtimescene.ts

```diff
@@ -862,11 +908,56 @@ namespace gdjs {
             this._variablesByExtensionName.get(extensionName);
           if (extensionVariables) {
             extensionVariables.updateFromNetworkSyncData(
-              extensionVariablesData
+              extensionVariablesData,
+              options
             );
           }
         }
       }
+      if (syncData.time) {
+        this._timeManager.updateFromNetworkSyncData(syncData.time);
+      }
+      if (syncData.layers) {
+        for (const layerName in syncData.layers) {
+          const layerData = syncData.layers[layerName];
+          if (this.hasLayer(layerName)) {
+            const layer = this.getLayer(layerName);
+            layer.updateFromNetworkSyncData(layerData);
+          }
+        }
+      }
+      if (syncData.once) {
+        this._onceTriggers.updateNetworkSyncData(syncData.once);
+      }
+      if (syncData.tween && gdjs.evtTools.tween.getTweensMap) {
+        gdjs.evtTools.tween.getTweensMap(this).updateFromNetworkSyncData(
```

Comment:
It's not clear why this code is so "complicated" in the runtime scene compared to the save:

<img width="498" height="108" alt="Image" src="https://github.com/user-attachments/assets/d9f36c3c-cd73-42fc-a171-5929d47cd0da" />

The load should basically be the same, and it's the tween extension job, or `updateFromNetworkSyncData` to do whatever it needs to pass functions to re-create the tween.

Reply:
done in callbacks for get & update 👍